src/pages/ShareCodePage.tsx

```diff
@@ -67,6 +81,21 @@ function ShareCodePage({report}: ShareCodePageProps) {
     const platform = getPlatform();
     const isNative = platform === CONST.PLATFORM.IOS || platform === CONST.PLATFORM.ANDROID;
 
+    const logo = isReport ? getLogoForWorkspace(policy) : (UserUtils.getAvatarUrl(currentUserPersonalDetails?.avatar, currentUserPersonalDetails?.accountID) as ImageSourcePropType);
+
+    // default logos (avatars) are SVG and because of that require some special logic
+    let logoSVG: React.FC<SvgProps> | undefined;
+    let logoBackground;
+    let logoColor;
+
+    if (policy && !policy.avatar) {
```

Comment:
Would that work?
```suggestion
    if (!policy?.avatar) {
```

Reply:
actually that does not work, because it will return true when `policy` is undefined, which is not my case.

I only want the specific case that:
there is a policy AND it has no avatar -> run default avatar logic